test/testEvents.test.js

```diff
@@ -0,0 +1,45 @@
+const Events = artifacts.require("Events");
+
+contract("Events", accounts => {
+    let [alice, bob] = accounts;
+    let contractInstance;
+    let eventNames = ["Mike's Birthday"]
+    let eventAttendance = 500;
+    
+    //Create new contract instance
+    beforeEach (async () => {
+        contractInstance = await Events.new();
+    });
+    //TODO Consider testing ownership
+    it("should create a new event", async () => {
+        const result = await contractInstance
+            .createEvent(eventNames[0], eventAttendance);
```

Comment:
Good opportunity here to test a variety of cases -- for example, I'm pretty sure this is going to break the code I wrote today because the apostrophes will screw up the JSON (I just have to URL encode the data before I store it, but that's a great thing to test for).